test cases/common/119 unresolved symbols/meson.build

```diff
@@ -0,0 +1,7 @@
+project('unresolved symbols', 'c')
+
+executable('unresolved-program', 'unresolved-symbol.c',
+  allow_undefined : true)
```

Comment:
You need to add `-DEXECUTABLE` here so that `main()` is defined. This is why the test fails:

```
[3/4] Linking target unresolved-program.exe
FAILED: unresolved-program.exe 
link @unresolved-program.exe.rsp
LINK : fatal error LNK1561: entry point must be defined
```